kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiCommands.java

```diff
@@ -560,4 +558,56 @@ public void run() {
       }
     }
   }
+
+  @Override
+  public String handleMultiLineCmd(String line) throws IOException {
```

Comment:
I believe you copy this method from the superclass w/ a little bit of modification, but it's hard to distinguish what's exactly modified here. How about making a prepared PR to copy the original method from the superclass? This could benefit reviewers.

Reply:
reverted changes by https://github.com/apache/incubator-kyuubi/pull/3762/commits/1c1cf31d2b5e296ef4df7307433880f484800c8a and filed a new issue #3770 for tracking this

Reply:
addressed in https://github.com/apache/incubator-kyuubi/pull/3762/commits/3e4d6e3fc05c641e519ac41d2330bc05dfb61d7f

Reply:
Shall we add unspported multi-line scripts to limitation in this PR description?

Reply:
it's supported now